app/views/energy_tariffs/energy_tariff_flat_prices/_form.html.erb

```diff
@@ -9,7 +9,7 @@
     </td>
     <td class="value align-middle">
       <%= f.input :value, pattern: '[0-9.]+', label: false,
-        input_html: { value: value_allowing_for_errors(energy_tariff_price) },
+        input_html: { value: value_allowing_for_errors(energy_tariff_price), style:"width: 8em" },
```

Comment:
perhaps this could be added as a class to `app/assets/stylesheets/energy_tariffs.scss`?

Reply:
Yes good point @iant, thanks! Making a change to the step too so that the up/down arrows use pence increments.